tools/harness-thci/OpenThread.py

```diff
@@ -3123,23 +3124,26 @@ def __detectZephyr(self):
 
     def __discoverDeviceCapability(self):
         """Discover device capability according to version"""
-        if self.IsBorderRouter:
+        thver = self.__executeCommand('thread version')[0]
+        if thver in ['1.3', '4'] and not self.IsBorderRouter:
+            self.log("Setting capability of {}: (DevCapb.C_BR13 | DevCapb.C_Host13 | DevCapb.C_Comm)".format(self))
+            self.DeviceCapability = OT13_CAPBS
+        elif thver in ['1.3', '4'] and self.IsBorderRouter:
```

Comment:
```suggestion
        elif thver in ['1.3', '4'] and not self.IsBorderRouter:
```

Reply:
@edmont Ah, now I see the point of confusion.  The logic is correct.  The log message is the issue, I think.

Reply:
The log message here is now correct.